intel_extension_for_transformers/neural_chat/pipeline/plugins/retrieval/indexing/indexing.py

```diff
@@ -137,4 +139,16 @@ def KB_construct(self, input):
                 return document_store
             else:
                 print("There might be some errors, please wait and try again!")
-
+        
+    def load(self, input):
+        if self.retrieval_type == "dense":
+            embedding = HuggingFaceInstructEmbeddings(model_name=self.embedding_model)
+            vectordb = Chroma(persist_directory=self.persist_dir, embedding_function=embedding)
+        else:
+            if self.document_store == "inmemory":
+                vectordb = self.KB_construct(input)
+            else:
+                vectordb = ElasticsearchDocumentStore(host="localhost", index=self.index_name,
+                                                            port=9200, search_fields=["content", "title"])

```

Comment:
port 9200 is default for elasticsearch api, but what if port 9200 is occupy by other app?

Reply:
ok, i will consider to add parms on "host" and "port"